src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -138,9 +138,9 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     this.reloadData();
   }
 
-  // let the ErrorBoundary handle errors by raising it
+  // Do not show the component if there is an error from the endpoint.
   renderError(): React.ReactNode {
```

Comment:
@NisanthanNanthakumar Are we still going to send the error to Sentry? Having it fail silently to the user seems OK but I think these unexpected errors should still go to Sentry.

Reply:
^ I agree, probs good to still get the errors in sentry

Reply:
when we set it to null, it will still collect the errors in sentry

Reply:
@NisanthanNanthakumar hold on, if you don't throw an error here it won't reach the error boundary. What you need to do is set the `customComponent` to `null` for the `ErrorBoundary`. See here:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx#L226-L228